x11-plugins/gkrellm-leds/gkrellm-leds-0.8.2-r2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gkrellm-plugin autotools
+
+MY_P="${P/rellm-/}"
+
+DESCRIPTION="GKrellM2 plugin for monitoring keyboard LEDs"
+HOMEPAGE="http://heim.ifi.uio.no/~oyvinha/gkleds/"
+SRC_URI="http://heim.ifi.uio.no/~oyvinha/e107_files/downloads/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~ppc ~sparc ~x86"
+IUSE=""
+
+RDEPEND="
+	app-admin/gkrellm:2[X]
+	x11-libs/libXtst"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=( "${FILESDIR}/${P}-r2-ldflags-typo.patch" )
```

Comment:
I'm not entirely sure which variables I should use in reference to the patch. My understanding so far is that one should refrain from `${PF}`, but the PMS lists several others (including `${P}` here) that aren't "consistent".

Reply:
I _think_ it's "not consistent" in the sense that "it might be different in e.g. pkg_postrm". It's safe to use it in PATCHES (but don't use ${PF} for the reasons @ionenwks mentioned in #gentoo-dev earlier; we generally anticipate we can revbump packages en-masse without affecting their installability. Using ${PF} harms that).

Reply:
Typically:
- Patch will likely keep across versions: ${PN}-0.0.0-name.patch (i.e. long standing issues, or Gentoo specific patches)
- Patch that should be either reviewed or dropped next version: ${P}-name.patch (i.e may be fixed / upstreamed, if not, can change to `${PN}` on bump)

And yeah, `${PF}` is just different given it includes revision making it too volatile over nothing. It shouldn't be thrown around casually and revbumps with no changes shouldn't be able to break things, *normally* only used for docs dir.

Reply:
OK, seems like I was right to change `${PF}` to `${P}` here. I included `-r2` in the patch name, because I wanted to reflect that it's been created for that revision. But it should probably also apply to any potential future revisions of this version.

Thanks for the input!